pkg/ConjuntoAnalisis.go

```diff
@@ -0,0 +1,5 @@
+package pkg
+
+type ConjuntoAnalisis struct{
```

Comment:
```suggestion
type ConjuntoArticulos struct{
```
Me parece que así se entendería mejor lo que hace cada clase